Source/SLSC Switch/SLSC Module SLSC Switch.xml

```diff
@@ -68,10 +68,10 @@
         <loc>SLSC Switch</loc>
       </Name>
       <GUID>5e47c8de-3ec6-490b-8897-5526d4aea756</GUID>
-      <Glyph>
-        <Type>To Application Data Dir</Type>
-        <Path>System Explorer\Glyphs\default fpga category.png</Path>
-      </Glyph>
+			<Glyph>
+				<Type>To Common Doc Dir</Type>
+				<Path>SLSC Plugins\Modules\SLSC EDS\Glyphs\SLSC_Module.png</Path>
+			</Glyph>
```

Comment:
This file is a mix of tabs and spaces; pick one.

Reply:
Spaces win.  I'll add a subset of the SLSC glyphs.